src/styles/theme/themes.ts

```diff
@@ -1,16 +1,34 @@
 import { IDefaultThemeProps, IBreakpoints, IThemeProps } from "./";
 
-const breakpoints: IBreakpoints = {
+const breakpoints = {
   breakpoints: {
```

Comment:
Aqui também será um objeto mais simples

```suggestion
const breakpoints = {
```